packages/turbo-repository/__tests__/detect.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from "node:path";
+import { Repository } from "../js/dist/index.js";
+
+describe("detect", () => {
```

Comment:
If it's helpful in the future we have a lot of testing utils in https://github.com/vercel/turbo/tree/main/packages/turbo-test-utils/src that we use throughout the JS packages to make creating, using, and tearing down fixtures really simple.